src/hooks/use-params.ts

```diff
@@ -11,6 +11,8 @@ interface Params {
 
   // url has "sol=y" meaning we should show option to start either solana or ethereum call
   isSolana: boolean;
+  // url has "sol=y" meaning we should show option for allowing address
+  isAllow: boolean;
```

Comment:
```suggestion
  // url has "allow=y" meaning we should show option for allowing address
```

Though perhaps naming the variable better (e.g. to `isAllowAddress` would eliminate the need for the comment).  Then it would not match the query string, but in fact `allow` is a fairly meaningless name for the query string too. So perhaps that should be `allow_address_exclusion` and the variable be `isAllowAddressExclusion`?